src/System.Formats.Tar.TestData/unarchived/many_small_files/9/8.txt

```diff
@@ -0,0 +1 @@
+Hello many_small_files
```

Comment:
Are there any behaviors of the implementation where having behavior needs to be tested for larger files, different content types, newlines, non-ascii characters or other encoding scenarios, or anything else like that?

Reply:
The contents of the file don't really matter much. The header stores the length in bytes of the data section, and the data section can contain anything in that space of the specified size.

But I can add tests (no need to add test assets) that verify that string fields do not have forbidden characters. For example, the Name field should not have an endline.